2019/en/src/0xb0-next-devs.md

```diff
@@ -15,7 +15,7 @@ comprehensive list of available projects.
 
 | | |
 |-|-|
-| **Education** | You can start reading [OWASP Education Project materials][2] according to your profession and interest. For hands-on learning, we added **crAPI** - **C**ompletely **R**idiculous **API** on [our roadmap][3]. Meanwhile, you can practice WebAppSec using the [OWASP DevSlop Pixi Module][4], a vulnerable WebApp and API service intent to teach users how to test modern web applications and API's for security issues, and how to write more secure API's in the future. You can also attend [OWASP AppSec Conference][5] training sessions, or [join your local chapter][6]. |
+| **Education** | You can start reading [OWASP Education Project materials][2] according to your profession and interest. For hands-on learning, we added **crAPI** - **C**ompletely **R**idiculous **API** on [our roadmap][3]. Meanwhile, you can practice WebAppSec using the [OWASP DevSlop Pixi Module][4], a vulnerable WebApp and API service intent to teach users how to test modern web applications and API's for security issues, and how to write more secure API's in the future. You can also attend [OWASP AppSec Conference][5] training sessions, or [join your local chapter][6]. Read about the latest API vulnerabilities and breaches at [APIsecurity.io][15]|
```

Comment:
Following [OWASP core values][1] and [principles][2], we should keep this document "vendor neutral" and "not driven by commercial interests".

Although **I** recognize the importance of apisecurity.io to bring awareness for API security issues, as it was done before ([140bb59][3], [2665cb3][4], [a9a5fb1][5]) this reference should be removed.

[1]: https://www.owasp.org/index.php/About_The_Open_Web_Application_Security_Project#Core_Values
[2]: https://www.owasp.org/index.php/About_The_Open_Web_Application_Security_Project#Principles
[3]: https://github.com/OWASP/API-Security/commit/140bb5942b54359ea7a9cd7e0ee8b788b0e4401b
[4]: https://github.com/OWASP/API-Security/commit/2665cb3d74894bf727c14a383e9c829e9ebef4fb#diff-f0514784b623a6250eede85cc3aa5226
[5]: https://github.com/OWASP/API-Security/commit/a9a5fb171e39914fb028b7d7b35f3dc62447a18d

Reply:
Makes sense. OK to remove.